main.ts

```diff
@@ -135,104 +288,161 @@ class PasteTransformSettingsTab extends PluginSettingTab {
 
 		containerEl.empty();
 
-		let patternsTa: TextAreaComponent | null = null;
-		let replacersTa: TextAreaComponent | null = null;
+		// Create a top-level container for our plugin to prevent CSS conflicts
+		const topLevelContainer = containerEl.createDiv({cls: 'paste-code-transform'});
```

Comment:
"Let's show a warning on the scripts page. Something like:

WARNING: Scripts have full access to your computer. Use only trusted scripts and proceed only if you fully understand what you're doing."